event_listener/dfusion_db/event_pusher.py

```diff
@@ -0,0 +1,43 @@
+from pymongo import MongoClient
+from django.conf import settings
+
+client = MongoClient(
+    host=settings.MONGO_HOST,
+    port=settings.MONGO_PORT
+)
+db = client.get_database(settings.DB_NAME)
+
+
+def post_deposit(event: dict):
```

Comment:
Why did you chose to expose this as global methods rather than a class? Given the name I was expecting a `EventPusher` class.

Reply:
I did this to avoid continually reinstatiating connections to the DB. If I move the connection factory into a class, it would be connecting on each event. I had considered a class, but thought this would be more valuable when there are more different type of posts being made. Anyway, I can start changing it today.